tcms_tenants/templates/tcms_tenants/new.html

```diff
@@ -43,7 +43,9 @@
                                {% if form.schema_name.field.disabled %}disabled{% endif %}>
                         <span class="input-group-addon">.{{ tcms_tenants_domain }}</span>
                         <span class="input-group-addon">
-                            {% attachment_form object%}
+                            {% if tenant_defined %}
```

Comment:
this is redundant b/c you have the `object` variable. It is either None or an instance.